src/silicone/database_crunchers/rms_closest.py

```diff
@@ -222,3 +224,30 @@ def _select_closest(to_search_df, target_series):
     labels, rmss = list(zip(*closeness))
     to_return = rmss.index(min(rmss))
     return dict(zip(to_search_df.index.names, labels[to_return]))
+
+
+def filter_for_overlap(df1, df2, cols):
```

Comment:
```suggestion
def _filter_for_overlap(df1, df2, cols):
```

Or let's add a test of `filter_for_overlap` explicitly.